docs/src/content/docs/ko/guides/css-and-tailwind.mdx

```diff
@@ -0,0 +1,284 @@
+---
+title: CSS & 스타일링
+description: 맞춤형 CSS로 Starlight 사이트의 스타일을 지정하거나 Tailwind CSS와 통합하는 방법을 알아보세요.
+---
+
+사용자 정의 CSS 파일을 사용하여 Starlight 사이트의 스타일을 지정하거나 Starlight Tailwind 플러그인을 사용할 수 있습니다.
+
+## 사용자 정의 CSS 스타일
+
+Starlight의 기본 스타일을 수정하거나 확장하기 위한 추가 CSS 파일을 제공하여 Starlight 사이트에 적용된 스타일을 사용자 정의하세요.
+
+1. `src/` 디렉토리에 CSS 파일을 추가합니다. 예를 들어, 페이지 제목에 대해 더 넓은 기본 열의 너비와 더 큰 텍스트 크기를 설정할 수 있습니다.
+
+   ```css
+   /* src/styles/custom.css */
+   :root {
+   	--sl-content-width: 50rem;
+   	--sl-text-5xl: 3.5rem;
+   }
+   ```
+
+2. `astro.config.mjs`에 있는 Starlight의 `customCss` 배열에 CSS 파일 경로를 추가하세요.
+
+   ```js
+   // astro.config.mjs
+   import { defineConfig } from 'astro/config';
+   import starlight from '@astrojs/starlight';
+
+   export default defineConfig({
+   	integrations: [
+   		starlight({
+   			title: 'Docs With Custom CSS',
```

Comment:
We could probably also translate the title text here like you did for the Tailwind example below.

Reply:
Thanks for the good point!